### ISSUE TITLE
Batch normalization is not calculating batch-wise mean during training in the following tutorial

### ISSUE THREAD

**Issue by ypxie** [tensorflow/hub#239]
https://github.com/tensorflow/hub/blob/master/examples/colab/image_feature_vector.ipynb

in the Build the model section:
```
# Load a pre-trained TF-Hub module for extracting features from images. We've
# chosen this particular module for speed, but many other choices are available.
image_module = hub.Module('https://tfhub.dev/google/imagenet/mobilenet_v2_035_128/feature_vector/2')
```
should't it be:

```
image_module = hub.Module('https://tfhub.dev/google/imagenet/mobilenet_v2_035_128/feature_vector/2',
tag = ["training"]
)

```
?

**Comment by andresusanopinto** [comment]
Note that the tutorial is using the module only as a feature extractor. As so it is using the inference graph and using the learned batch normalization factors (not performing per batch normalization).

**Comment by ypxie** [comment]
Thanks for the explanation, but if we load the model as trainable, shouldn't we also use batch mean instead of the pre-computed mean?
